ewlegal.cls

```diff
@@ -0,0 +1,84 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+%% Work consists of the files engwales-legal.cls and accompanying engwales-*.sty
+
+
+\NeedsTeXFormat{LaTeX2e}
+\ProvidesClass{ewlegal}[2021-02-21 Legal Templates for England and Wales]
+
+%% Uses article as a base
+\LoadClass[12pt,a4paper]{article}% TODO Perhaps use KOMA-Script?
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Options
+%%%%%
+
+%% Easy option management
+\RequirePackage{kvoptions}
+
+%% Select the template used
+% TODO Be able to nicely handle multiple templates being selected
+
+\DeclareBoolOption[false]{pleading}
+
+\DeclareBoolOption[false]{opinion}
```

Comment:
what happens (or _should_ happen) if non were selected?

Reply:
I don't know... I was having some issues with it playing nice unless it was marked as false by default. I presume it doesn't load a specific format so you just get an article without many frills, and no reworked title. If @passamo9, or someone else thinks one should be a default then we can work that in.

Reply:
or require at least one of the options to be set. at that point maybe `type=opinion` than individual options?